libcloudforensics/providers/gcp/internal/compute_base_resource.py

```diff
@@ -65,13 +65,13 @@ def FormatLogMessage(self, message: str) -> str:
 
     return 'project:{0} {1}'.format(self.project_id, message)
 
-  def GetOperation(self) -> Dict[str, Any]:
+  def GetOperation(self) -> 'Dict[str, Any]':
```

Comment:
Why are we using quotes here? The other `Dict` type checks (see L39) don't have them

Reply:
removed